hooks/post/10-post_install.rb

```diff
@@ -44,23 +44,22 @@ def new_install?
         foreman_oauth_key     = Kafo::Helpers.read_cache_data("oauth_consumer_key")
         foreman_oauth_secret  = Kafo::Helpers.read_cache_data("oauth_consumer_secret")
         katello_oauth_secret  = Kafo::Helpers.read_cache_data("katello_oauth_secret")
-        org                   = @kafo.param('certs', 'org').value
         say <<MSG
 
   To finish the installation, follow these steps:
 
-  If you do not have the smartproxy registered to the Katello instance, then please do the following:
-
-  1. yum -y localinstall http://#{fqdn}/pub/katello-ca-consumer-latest.noarch.rpm
-  2. subscription-manager register --org "<%= color('#{org}', :info) %>"
-
-  Once this is completed run the steps below to start the smartproxy installation:
-
   1. Ensure that the foreman-installer-katello package is installed on the system.
```

Comment:
@ehelms I put the `installer_name` back, but this package name is also going to cause a problem.  This would need to say `satellite-installer` downstream